casadm/cas_main.c

```diff
@@ -334,9 +344,33 @@ int handle_start()
 		return FAILURE;
 	}
 
-	if (command_args_values.state == CACHE_INIT_STANDBY && command_args_values.force) {
-		cas_printf(LOG_ERR, "Use of 'failover-standby' and 'force' simultaneously is forbidden.\n");
-		return FAILURE;
+	if (command_args_values.state == CACHE_INIT_STANDBY) {
+		if (command_args_values.force) {
+			cas_printf(LOG_ERR, "Use of 'failover-standby' and 'force' simultaneously is forbidden.\n");
+			return FAILURE;
+		}
+		if (command_args_values.cache_id == 0) {
+			cas_printf(LOG_ERR, "Use of 'failover-standby' without 'cache-id' explicitly specified is forbidden.\n");
```

Comment:
What we want to say to user is rather something like "'failover-standby' requires specifying 'cache-id'". 
That would be a meaningful hint.